packages/types/src/project.ts

```diff
@@ -22,108 +60,276 @@ export type RuntimeHandlerInputMap<T extends AnyTuple = AnyTuple> = {
 };
 
 type RuntimeFilterMap = {
-  [SubstrateHandlerKind.Block]: SubstrateNetworkFilter;
+  [SubstrateHandlerKind.Block]: SubstrateBlockFilter;
   [SubstrateHandlerKind.Event]: SubstrateEventFilter;
   [SubstrateHandlerKind.Call]: SubstrateCallFilter;
 };
 
-export interface ProjectManifest {
-  specVersion: string;
-  description: string;
-  repository: string;
-
-  schema: string;
-
-  network: {
-    endpoint: string;
-    customTypes?: RegistryTypes;
-  };
-
-  bypassBlocks?: number[];
-  dataSources: SubstrateDatasource[];
-}
-
 // [startSpecVersion?, endSpecVersion?] closed range
 export type SpecVersionRange = [number, number];
 
 interface SubstrateBaseHandlerFilter {
   specVersion?: SpecVersionRange;
 }
 
+/**
+ * Represents a filter for Substrate blocks, extending SubstrateBaseHandlerFilter.
+ * @interface
+ * @extends {SubstrateBaseHandlerFilter}
+ */
 export interface SubstrateBlockFilter extends SubstrateBaseHandlerFilter {
+  /**
+   * The modulo value for filtering blocks (optional).
+   * @type {number}
+   * @example
+   * modulo: 5, // every 5 blocks will be indexed
+   */
   modulo?: number;
+
+  /**
+   * A cron expression will index blocks at an interval (optional).
+   * @type {string}
+   * @example
+   * timestamp: "\*\/5 * * * *"
+   */
   timestamp?: string;
 }
 
+/**
+ * Represents a filter for Substrate events, extending SubstrateBaseHandlerFilter.
+ * @interface
+ * @extends {SubstrateBaseHandlerFilter}
+ */
 export interface SubstrateEventFilter extends SubstrateBaseHandlerFilter {
+  /**
+   * The module name for filtering events or calls (optional).
+   * @type {string}
+   * @example
+   * module: 'balances'
+   */
   module?: string;
+
+  /**
+   * The method name for filtering events calls (optional).
```

Comment:
Is this case sensitive? If so
```suggestion
   * The method name for filtering events calls (case-insensitive) (optional).
```